osu.Game/Overlays/Profile/Header/Components/DailyChallengeStatsDisplay.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.LocalisationExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Resources.Localisation.Web;
+using osu.Game.Scoring;
+
+namespace osu.Game.Overlays.Profile.Header.Components
+{
+    public partial class DailyChallengeStatsDisplay : CompositeDrawable, IHasCustomTooltip<DailyChallengeTooltipData>
+    {
+        public readonly Bindable<UserProfileData?> User = new Bindable<UserProfileData?>();
+
+        public DailyChallengeTooltipData? TooltipContent { get; private set; }
+
+        private OsuSpriteText dailyPlayCount = null!;
+
+        [Resolved]
+        private OsuColour colours { get; set; } = null!;
+
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; } = null!;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            AutoSizeAxes = Axes.Both;
+            CornerRadius = 5;
+            Masking = true;
+
+            InternalChildren = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colourProvider.Background4,
+                },
+                new FillFlowContainer
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Padding = new MarginPadding(5f),
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Horizontal,
+                    Children = new Drawable[]
+                    {
+                        new OsuTextFlowContainer(s => s.Font = OsuFont.GetFont(size: 12))
+                        {
+                            AutoSizeAxes = Axes.Both,
+                            Text = UsersStrings.ShowDailyChallengeTitle,
```

Comment:
Hmm, this wasn't checked:
<img width="180" alt="Screenshot 2024-08-02 at 10 36 31 AM" src="https://github.com/user-attachments/assets/1d8bbfdc-a0b2-408e-bf5a-ea65d5302ce7">

Seems like we need to reconsider the `\n`. The fallback uses verbatim `@`, and it wasn't the first time we've seen this: https://github.com/ppy/osu/pull/23049

Reply:
Fwiw including the `\n` verbatim was intentional from my side. It is a single-quoted string in the localisation and so it is expected to arrive verbatim, and osu-web handles it specially: https://github.com/ppy/osu-web/blob/53931864ee65917765df1e2d7244dac68ca93782/resources/js/profile-page/daily-challenge.tsx#L129-L131

Reply:
How do we handle this locally? I'm not sure it's possible.

Reply:
I think we can using a custom `ILocalisableStringData` implementation that accepts the translation and interpret the \n as a new line. I'll look into it.

Reply:
It would be something similar to [CaseTransformableString](https://github.com/ppy/osu-framework/blob/master/osu.Framework/Localisation/CaseTransformableString.cs)

We really should've made a `TransformableString` or something that can take an arbitrary `Func<>`.

Reply:
I'm just going to undo the localiasation and merge. This is all a PITA and I'm not willing to try and fix it right now. We can just add a second localisation if we want, rather than fucking around with web being unwilling to change the localisation.